cmd/ebrelayer/main.go

```diff
@@ -67,27 +70,43 @@ var rootCmd = &cobra.Command{
 	SilenceUsage: true,
 }
 
-//	initRelayerCmd : Initializes a relayer service run by individual
-//		validators which streams live events from a smart contract.
+//	initEthereumRelayerCmd : Initializes a relayer service run by individual
+//		validators which streams live events from an Ethereum smart contract.
 //		The service automatically signs messages containing the event
 //		data and relays them to tendermint for handling by the
 //		EthBridge module.
 //
-func initRelayerCmd() *cobra.Command {
-	initRelayerCmd := &cobra.Command{
-		Use:   "init [web3Provider] [contractAddress] [eventSignature] [validatorFromName] --chain-id [chain-id]",
-		Short: "Initializes a web socket which streams live events from a smart contract",
+func initEthereumRelayerCmd() *cobra.Command {
+	initEthereumRelayerCmd := &cobra.Command{
+		Use:   "initEth [web3Provider] [contractAddress] [eventSignature] [validatorFromName] --chain-id [chain-id]",
```

Comment:
you could maybe create an `init` Command and then use `AddCommand` to add two separate `ethereum` and `cosmos` commands. IMO it'd provide a better UX.

Reply:
```suggestion
		Use:   "ethereum [web3Provider] [contractAddress] [eventSignature] [validatorFromName] --chain-id [chain-id]",
```

Reply:
Agreed, this UX is much better. Added the `init` subcommand in https://github.com/cosmos/peggy/pull/77/commits/eea99956698726a920b255fd281efa1eda41e914